manual/manual/library/stdlib.etex

```diff
@@ -99,6 +99,10 @@ be called from C \\
 "Spacetime" & p.~\pageref{Spacetime} & memory profiler \\
 "Sys" & p.~\pageref{Sys} & system interface \\
 \end{tabular}
+\subsubsection{Misc:}
+\begin{tabular}{lll}
+"Fun" & p.~\pageref{Fun} & operators on functional values \\
+\end{tabular}
 \end{latexonly}
 
 \ifouthtml
```

Comment:
Two lists of modules follow this directive you should add the `Fun` module there too in alphabetic order (this is unfortunately not checked by the check manual script /cc @Octachron)

Reply:
Thanks!  One should add a guide on how to add modules to the stdlib (e.g. in CONTRIBUTING.md, section "Contributing to the standard library"), since this requires advanced skills .

Reply:
Actually there are some instructions [here](https://github.com/ocaml/ocaml/blob/trunk/stdlib/HACKING.adoc) but they seem to be incomplete.